docs/repositories.md

```diff
@@ -39,9 +39,13 @@ poetry source add foo https://pypi.example.org/simple/
 Then, assuming the repository requires authentication, configure credentials for it.
 
 ```bash
-poetry config http-basic.foo username password
+poetry config http-basic.foo <username> <password>
 ```
 
+{{% warning %}}
+Depending on your system's configuration, credentials might be saved in your command line history.
```

Comment:
Could be worth mentioning that many shells do not save space-prefixed commands into history. It's not universal, but it's a very common feature for this use case.